content/doc/book/system-administration/administering-jenkins-on-kubernetes.adoc

```diff
@@ -12,4 +12,176 @@ ifdef::backend-html5[]
 :toc: left
 endif::[]
 
-= Administering Jenkins on Kubernetes
\ No newline at end of file
+= Administering Jenkins on Kubernetes
+
+In this section, we will look at:
+
+* Upgrading from one version of Kubernetes to another.
+* How to upgrade the version of Kubernetes in your cluster.
+* Upgrade Kubernetes cluster maintained by the Jenkins Infrastructure project.
```

Comment:
This should be a link to the Jenkins infrastructure documentation if it is included in this document at all.  We don't want to duplicate the infra team's documentation because then it needs to be maintained in two places.
```suggestion
```